pandas/tests/generic/test_generic.py

```diff
@@ -548,9 +548,6 @@ def test_validate_bool_args(self):
             with pytest.raises(ValueError):
                 super(DataFrame, df).drop("a", axis=1, inplace=value)
 
-            with pytest.raises(ValueError):
-                super(DataFrame, df).sort_index(inplace=value)
-
```

Comment:
Why is this being dropped?

Reply:
I guess this would technically raise an AbstractMethodError now; indifferent on whether should update the error or keep removed

But would be nice to update this test for Python3 super calls and parametrization